src/storage/migration.rs

```diff
@@ -216,6 +221,20 @@ fn migrate_0_19_to_0_21_0(db: &Database) -> Result<()> {
     Ok(())
 }
 
+#[allow(clippy::too_many_lines)]
```

Comment:
Could you please remove the `#[allow(clippy::too_many_lines)]`?

Reply:
The change has been completed.
Thank you.